test/WorkerExtensionTests/Blob/BlobStorageConverterTests.cs

```diff
@@ -30,8 +30,8 @@ namespace Microsoft.Azure.Functions.WorkerExtension.Tests
 {
```

Comment:
There are a couple tests that are skipped here, can you enable those and see if they work now? A couple of them had an issue because wee were using `Convert.ChangeType` and that's gone now so I'm hoping those tests pass